Packs/CommonPlaybooks/Playbooks/playbook-IP_Enrichment_-_External_-_Generic_v2.yml

```diff
@@ -601,6 +669,47 @@ tasks:
     note: false
     timertriggers: []
     ignoreworker: false
+    continueonerrortype: ""
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "44":
+    id: "44"
+    taskid: 3607b68d-96db-4ed1-8324-e21009548cf8
+    type: regular
+    task:
+      id: 3607b68d-96db-4ed1-8324-e21009548cf8
+      version: -1
+      name: Check the IP reputation
+      description: Checks the reputation of an IP address.
+      script: VirusTotal (API v3)|||ip
```

Comment:
Let's use` !IP` (not only using VT)

Reply:
Done.
The task was changed and now it uses "!IP" automation.
Additionally, I separated the task from the VirusTotal section flow and added another section flow named "Check Reputation Using All Available Integrations"